css/tabstrip.css

```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
 .tab.selected .tab-thumbnail {
   transform: scale(1);
 }
+
+.tab-close-button {
+  font-family: FontAwesome;
+  text-align: center;
+  font-size: 16px;
+  vertical-align: middle;
+  width: 22px;
+  height: 22px;
+  position: absolute;
+  top: 10px;
+  right: -10px;
```

Comment:
It's a square, vertically and horizontally aligned. I would expect `top = -right`.